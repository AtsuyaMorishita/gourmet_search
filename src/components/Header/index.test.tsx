import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import { Header } from ".";

describe("header", () => {
  it("render correctly", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
