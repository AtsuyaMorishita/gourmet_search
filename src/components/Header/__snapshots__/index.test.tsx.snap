// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header render correctly 1`] = `
<div>
  <header
    class="text-center border-b  fixed w-[100%] top-0 bg-white z-20"
  >
    <div
      class="flex justify-between items-center px-4 lg:px-6"
    >
      <h1
        class="inline-block py-3"
      >
        <a
          class="text-md font-bold"
          href="/"
        >
          <span
            class="text-[#017D01] text-xl"
          >
            グルメ検索
          </span>
          アプリ
        </a>
      </h1>
      <div
        class="flex gap-4"
      />
    </div>
  </header>
</div>
`;
